sdk/batch/azure-compute-batch/src/test/java/com/azure/compute/batch/JobScheduleTests.java

```diff
@@ -73,22 +75,35 @@ public void canCRUDJobSchedule() throws Exception {
             }
             Assert.assertTrue(found);
 
+            // REPLACE
+            List<MetadataItem> metadataList = new ArrayList<>();
+            metadataList.add(new MetadataItem("name1", "value1"));
+            metadataList.add(new MetadataItem("name2", "value2"));
+
+            jobSchedule.setMetadata(metadataList);
+            batchClient.replaceJobSchedule(jobScheduleId, jobSchedule);
+
+            List<MetadataItem> retrievedMetadata = jobSchedule.getMetadata();
```

Comment:
I would reinitiate a GET request on the JobSchedule after the REPLACE/PUT request and then assert the metadata.